src/components/DefaultCity.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+//redux
+import { useSelector } from "react-redux";
+//style
+import styled from "styled-components";
+
+import { findAverage } from "../util";
+
+const DefaultCity = () => {
+  //redux
+  const {
+    defaultCity,
+    currentWeather,
+    isLoading,
+    defaultForecastTemp,
+    defaultForecastPressure,
+    defaultForecastHumidity,
+  } = useSelector((state) => state.default);
+
+  //function to get icon url for default city
+  const icon = () => {
+    let iconURL;
+    if (!isLoading) {
+      iconURL = `http://openweathermap.org/img/wn/${currentWeather}.png`;
+    }
+    return iconURL;
+  };
+
+  return (
+    <StyledDefaultCity>
+      {!isLoading && (
+        <StyledCurrentWeather>
+          <div className="weather-details">
+            <h3>{defaultCity}</h3>
+            <p>{findAverage(defaultForecastTemp)}F</p>
```

Comment:
I would consider putting this logic inside the reducer since I don't believe you're using the forecast data for anything other than this (though I may be being hypocritical here as we may have done something similar somewhere in the curriculum, though I don't recall)